### PR TITLE
Fix disputes related wording.

### LINKED ISSUE #6801: Adjust wording of dispute notice on WC core order page
Starting in 6.2.0, we now [alert merchants on the WC core order page](https://github.com/Automattic/woocommerce-payments/blob/6.2.0-test-3-bis/client/order/index.js#L162) when an order's transaction has been disputed. An example alert:

> This order has a chargeback dispute of $50.00 labeled as "Transaction unauthorized". Please respond to this dispute before Jul 26, 2023.

However, we do not refer to disputes as "chargebacks", and a "chargeback dispute" is an awkward phrasing anyway. Can we change this to the following?

> This order has been disputed in the amount of `$amount`. The customer provided the following reason: `$reason`. Please respond to this dispute before `$deadline`.

cc @souravdebnath1986 @Jinksi

### LINKED ISSUE #6802: Change dispute alert wording for transactions with an inquiry
Starting in 6.2.0, we now [alert merchants on the WC core order page](https://github.com/Automattic/woocommerce-payments/blob/6.2.0-test-3-bis/client/order/index.js#L162) when an order's transaction has been disputed. An example alert:

> This order has a chargeback dispute of $50.00 labeled as "Transaction unauthorized". Please respond to this dispute before Jul 26, 2023.

However, we also show this same wording for payments against which [an inquiry](https://stripe.com/docs/disputes/how-disputes-work#inquiries) has been filed. As Stripe says, an inquiry is "a preliminary phase before creating a formal dispute", so we should avoid calling them disputes if possible.

Can we change the alert wording for inquiries to the following?

> The card network involved in this order has opened an inquiry into the transaction with the following reason: $reason. Please respond to this inquiry before $deadline, just like you would for a formal dispute.

### LINKED ISSUE #6803: Avoid using the word "chargeback" in settings explainer text
On the Settings page, we use [this sentence](https://github.com/Automattic/woocommerce-payments/blob/develop/client/settings/settings-manager/index.js#L123) to explain the fraud and risk tools settings:

> Help avoid chargebacks by setting your security and fraud protection risk level.

However, we should avoid using the word "chargeback" if possible, since we've more or less settled on calling them "disputes".

Can we please change this to something like the following?

> Help avoid unauthorized transactions and disputes by setting your fraud protection level.

### PR DESCRIPTION
**Pull request by brucealdridge**
Fixes #6803
Fixes #6802 
Fixes #6801 

#### Changes proposed in this Pull Request

Dispute order notices wording has been updated for all inquiries. Previously the notices incorrectly referred to inquiries as disputes. Dispute notices also included the word `chargeback` which is a synonym for dispute that we are not using in WooPayments. 

The wording and color of these notices change based on how much time is left to respond to the dispute. Screenshots are provided below to show the different states of both dispute and inquiry notices.

| | **Dispute** | **Inquiry** |
|--------|--------|--------|
| **7+ days left** | Screenshot 2023-08-10 at 4 43 28 PM  | Screenshot 2023-08-10 at 4 43 05 PM |
| **3-6 days left** | Screenshot 2023-08-10 at 4 43 51 PM | Screenshot 2023-08-10 at 4 42 42 PM |
| **1-2 days left** | Screenshot 2023-08-10 at 4 44 20 PM | Screenshot 2023-08-10 at 4 42 10 PM |
| **Last day** | Screenshot 2023-08-10 at 4 44 38 PM | Screenshot 2023-08-10 at 4 41 09 PM | 

The "Fraud Protection" section in `Payments > Settings` also mentioned `chargebacks` which has now been removed.

Screenshot 2023-08-10 at 4 46 37 PM

#### Testing instructions

**Fraud Protection Settings**
1. Visit `Payments > Settings`
2. Scroll down to "Fraud Protection Settings" 
3. See that the helper text does not mention "chargebacks"

**Inquiry**
1. Create a new order with an inquiry test card `4000000000001976`
2. Visit the order screen in WC Admin.
3. Verify the correct wording is visible in the dispute order notice. It should clearly state that there is an "Inquiry".

**Dispute**
1. Create a new order with an dispute test card `4000000000000259`
2. Visit the order screen in WC Admin.
3. Verify the correct wording is visible in the dispute order notice. There should be no mention of "chargeback".

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: client/order/index.js [hunk 1]
```diff
@@ -192,13 +204,19 @@ const DisputeNotice = ( { chargeId } ) => {
...
+		dateI18n( 'M j, Y', dueBy.local().toISOString() )
+	);
+
 	// If the dispute is due within 72 hours, we want to highlight it as urgent/red.
 	if ( countdownDays < 3 ) {
 		urgency = 'error';
 	}
 
 	if ( countdownDays < 1 ) {
-		urgency = 'error';
```
Comment (brucealdridge) [inline client/order/index.js:201]: Duplication: If less than 3 days, urgency already set above.

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6970` or branch name `fix/6803-disputes-wording` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=fix/6803-disputes-wording) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: dd7d529d4188077638170e4e7930b878a048dd9a
- Build time: 2023-08-14 00:06:36 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/disputes/utils.ts [hunk 2]
```diff
@@ -45,3 +49,8 @@ export const isDueWithin = ( { dueBy, days }: IsDueWithinProps ): boolean => {
 	const isPastDue = now.isAfter( dueByMoment );
 	return isWithinDays && ! isPastDue;
 };
+
+export const isInquiry = ( dispute: Dispute | CachedDispute ): boolean => {
+	// Inquiry disputes statuses are one of `warning_needs_response`, `warning_under_review` or `warning_closed`.
```
Comment (Jinksi) [inline client/disputes/utils.ts]: Super-minor nitpick:
```suggestion
	// Inquiry dispute statuses are one of `warning_needs_response`, `warning_under_review` or `warning_closed`.
```

File: client/disputes/utils.ts [hunk 3]
```diff
@@ -45,3 +49,8 @@ export const isDueWithin = ( { dueBy, days }: IsDueWithinProps ): boolean => {
...
+	return dispute.status.indexOf( 'warning' ) === 0;
```
Comment (Jinksi) [inline client/disputes/utils.ts]: No practical difference, but potentially a _slightly_ more readable alternative.

```suggestion
	return dispute.status.startsWith( 'warning' );
```

File: client/order/index.js [hunk 4]
```diff
@@ -17,6 +17,7 @@ import { formatExplicitCurrency } from 'utils/currency';
 import { reasons } from 'wcpay/disputes/strings';
 import { getDetailsURL } from 'wcpay/components/details-link';
 import { disputeAwaitingResponseStatuses } from 'wcpay/disputes/filters/config';
```
Comment (Jinksi) [inline client/order/index.js:19]: Not for this PR, but makes me think we could improve `disputeAwaitingResponseStatuses` logic by creating an `isAwaitingResponse` util function, similar to `isInquiry`.

File: client/order/index.js [hunk 5]
```diff
@@ -149,37 +150,48 @@ const DisputeNotice = ( { chargeId } ) => {
...
 	}
 
+	const titleStrings = {
+		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
+		dispute_default: __(
+			// eslint-disable-next-line max-len
+			'This order has been disputed in the amount of %1$s. The customer provided the following reason: %2$s. Please respond to this dispute before %3$s.',
+			'woocommerce-payments'
+		),
+		inquiry_default: __(
```
Comment (Jinksi) [inline client/order/index.js:161]: Missing translators comment. 
```suggestion
    // Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
	  inquiry_default: __(
```

**Review by Jinksi — APPROVED** [review]
LGTM @brucealdridge 🙌 I've made some suggestions, mostly minor, although the missing `Translators` comment should be resolved.

I've tested the following:
**Fraud Protection Settings** – text updated without reference to “chargebacks” ✅
**Inquiry** – text updated as per PR description for urgent and non-urgent `warning_` disputes ✅
**Dispute** – text updated as per PR description for urgent and non-urgent disputes ✅

Note that some of your screenshots are in the table rows above (e.g. Inquiry 7 days / 6 days are switched) but the code and outcome is correct.

Approved from my perspective with @aheckler's approval.

**Comment by brucealdridge** [comment]
Are you happy with these changes @aheckler ?

**Comment by brucealdridge** [comment]
Merging as these changes are significantly better than what is there. Also require `isInquiry` for #6998

**Pull request merged**